spec/controllers/new_gestionnaire/procedures_controller_spec.rb

```diff
@@ -336,4 +336,25 @@
       end
     end
   end
+
+  describe '#update_email_notifications' do
+    let(:gestionnaire) { create(:gestionnaire) }
+    let!(:procedure) { create(:procedure, gestionnaires: [gestionnaire]) }
+
+    context "when logged in" do
+      before { sign_in(gestionnaire) }
+
+      it { expect(procedure.gestionnaires_with_email_notifications).to match([]) }
```

Comment:
Dans ma tête, `match` sert à comparer deux tableaux sans prendre en compte l’ordre des éléments. Du coup, sur un tableau vide, ça revient au même qu’un simple `eq`.

Du coup, j’ai été surpris de trouver `match` ici plutôt que `eq`,, et j’ai préféré prendre le temps de relire la doc de `match` et faire quelques essais pour vérifier que ça n’avait pas un comportement spécial qui m’échappait. Conclusion : non, ça revient vraiment au même qu’un simple `eq` ici.

Du coup, je trouverais ça plus sympa pour la prochaine personne qui lira / modifiera ce test de mettre `expect(...).to eq([])`, ou même `expect(...).to be_empty`